src/Agent.Listener/Configuration/WindowsRegistryManager.cs

```diff
@@ -0,0 +1,90 @@
+// #if OS_WINDOWS
```

Comment:
sorry. missed reverting back the temp changes.

---
In reply to: [119686173](https://github.com/Microsoft/vsts-agent/pull/985#discussion_r119686173) [](ancestors = 119686173)